test/fw/ptl/utils/plugins/ptl_test_data.py

```diff
@@ -159,154 +157,41 @@ def __save_home(self, test, status, err=None):
-                f.write(_msg + '\n')
-                self.logger.error(_msg)
-                f.close()
-                return
-            self.du.rm(hostname=current_host, path=diag_out_dest,
-                       force=True, sudo=True, level=logging.DEBUG2)
-            diag_out_dest = diag_out_dest.rstrip('.tar')
-            for c in cores:
-                cmd = [pbs_diag, '-g', c]
-                ret = self.du.run_cmd(svr_host, cmd, sudo=True,
-                                      level=logging.DEBUG2)
-                if ret['rc'] != 0:
-                    _msg = 'Failed to get core file information for '
-                    _msg += '%s on %s:' % (c, svr_host)
-                    _msg += '\n\n' + '\n'.join(ret['err']) + '\n\n'
-                    f.write(_msg + '\n')
-                    self.logger.error(_msg)
-                else:
-                    of = os.path.join(diag_out_dest,
-                                      os.path.basename(c) + '.out')
-                    _f = open(of, 'w+')
-                    _f.write('\n'.join(ret['out']) + '\n')
-                    _f.close()
-                    self.du.rm(hostname=svr_host, path=c, force=True,
-                               sudo=True, level=logging.DEBUG2)
-            cmd = ['tar', '-cf', diag_out_dest + '.tar']
-            cmd += [os.path.basename(diag_out_dest)]
-            ret = self.du.run_cmd(current_host, cmd, sudo=True, cwd=datadir,
-                                  level=logging.DEBUG2)
-            if ret['rc'] != 0:
-                _msg = 'Failed generate tarball of diag directory'
-                _msg += ' %s' % diag_out_dest
-                _msg += ' after adding core(s) information in it:'
-                _msg += '\n\n' + '\n'.join(ret['err']) + '\n\n'
-                f.write(_msg + '\n')
-                self.logger.error(_msg)
-                f.close()
-                return
-            cmd = ['gzip', diag_out_dest + '.tar']
-            ret = self.du.run_cmd(current_host, cmd, sudo=True,
-                                  level=logging.DEBUG2)
-            if ret['rc'] != 0:
-                _msg = 'Failed compress tarball of diag %s' % diag_out_dest
-                _msg += '.tar after adding core(s) information in it:'
-                _msg += '\n\n' + '\n'.join(ret['err']) + '\n\n'
-                f.write(_msg + '\n')
-                self.logger.error(_msg)
-                f.close()
-                return
-            self.du.rm(current_host, diag_out_dest, sudo=True,
-                       recursive=True, force=True, level=logging.DEBUG2)
-        else:
-            diag_out_dest = diag_out_dest.rstrip('.tar.gz')
-        dest = os.path.join(datadir,
-                            'PBS_' + current_host.split('.')[0] + '.tar.gz')
-        ret = self.du.run_copy(current_host, diag_out_dest + '.tar.gz',
-                               dest, sudo=True, level=logging.DEBUG2)
-        if ret['rc'] != 0:
```

Comment:
The tarball was being renamed earlier to something else, is that not needed anymore?

Reply:
Nope... previously we were renaming tar to fulfill Altair's in-house tool requirement, now it's in-house tool's responsibility to take care of it. PTL will provide whatever snapshot provides.

If you still think that we should do it then I suggest to do it in pbs_snapshot (which will be new RFE for pbs_snapshot), not in PTLTestData as it is just using pbs_snapshot to get analysis data.